### ISSUE TITLE
Default configuration specified in README fails for android

### ISSUE THREAD

**Issue by jeremy-dobler** [intercom/intercom-cordova#228]
I attempted to add intercom to my app recently. 

Ran the following commands as specified in README

```
$ ionic cordova plugin add cordova-plugin-intercom
$ npm install --save @ionic-native/intercom
```

Then I added my intercom preferences to the config.xml which I got by logging into my Intercom account and grabbing from the App Settings area.
```
<preference name="intercom-app-id" value="my_key" />
<preference name="intercom-ios-api-key" value="ios_sdk-something" />
<preference name="intercom-android-api-key" value="android_sdk-something" />
```

I also added it to my app.module.ts and imported into my app.component.ts
```
platform.ready().then(() => {
    intercom.registerUnidentifiedUser({});
    ....
    ....
});
```

When I build the apk for android, running `ionic cordova build android`, the generated apk fails to load on a phone. I did some debugging and determined that it is erroring on the following lines of code in the io.intercom.android.sdk.IntercomBridge.java class

```
ApplicationInfo app = context.getPackageManager().getApplicationInfo(context.getPackageName(), PackageManager.GET_META_DATA);
Bundle bundle = app.metaData;

//Get app credentials from config.xml or the app bundle if they can't be found
String apiKey = IntercomBridge.this.preferences.getString("intercom-android-api-key", bundle.getString("intercom_api_key"));
String appId = IntercomBridge.this.preferences.getString("intercom-app-id", bundle.getString("intercom_app_id"));
```

Specifically it is failing because "bundle" is undefined. If I evaluate the code and run `IntercomBridge.this.preferences.getString("intercom-android-api-key", "blah");` then it gets back the correct keys. 

So my question is how do I make "bundle" defined in this context.

## Version info
  - cordova-plugin-intercom: 4.1.2
  - @ionic-native/intercom: 4.5.2
  - iOS/Android OS version: API 27 (Android Emulator)
  - iOS/Android Device: Nexus 5X (Android Emulator)
```
$ ionic info
cli packages: (C:\Users\jeremy.dobler\AppData\Roaming\nvm\v6.11.0\node_modules)
    @ionic/cli-utils  : 1.19.0
    ionic (Ionic CLI) : 3.19.0

global packages:
    cordova (Cordova CLI) : 7.1.0

local packages:
    @ionic/app-scripts : 3.1.7
    Cordova Platforms  : android 6.3.0
    Ionic Framework    : ionic-angular 3.9.2

System:
    Android SDK Tools : 26.1.1
    Node              : v6.11.0
    npm               : 3.10.10
    OS                : Windows 10

Environment Variables:
    ANDROID_HOME : C:\Users\...\AppData\Local\Android\Sdk

Misc:
    backend : pro
```

## Expected behavior
App should load with default configurations specified in README
## Actual behavior
App does not load and throws FatalException
```
FATAL EXCEPTION: main
                  Process: com.bullhorn.mobile, PID: 13399
                  java.lang.RuntimeException: Unable to start activity ComponentInfo{com.bullhorn.mobile/com.bullhorn.mobile.MainActivity}: java.lang.IllegalStateException: Please call Intercom.initialize() before requesting the client.
                      at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2778)
                      at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856)
                      at android.app.ActivityThread.-wrap11(Unknown Source:0)
                      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589)
                      at android.os.Handler.dispatchMessage(Handler.java:106)
                      at android.os.Looper.loop(Looper.java:164)
                      at android.app.ActivityThread.main(ActivityThread.java:6494)
                      at java.lang.reflect.Method.invoke(Native Method)
                      at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
                   Caused by: java.lang.IllegalStateException: Please call Intercom.initialize() before requesting the client.
                      at io.intercom.android.sdk.Intercom.client(Intercom.java:115)
                      at io.intercom.android.sdk.IntercomBridge$2.run(IntercomBridge.java:58)
                      at android.app.Activity.runOnUiThread(Activity.java:6174)
                      at io.intercom.android.sdk.IntercomBridge.onStart(IntercomBridge.java:53)
                      at org.apache.cordova.PluginManager.onStart(PluginManager.java:275)
                      at org.apache.cordova.CordovaWebViewImpl.handleStart(CordovaWebViewImpl.java:462)
                      at org.apache.cordova.CordovaActivity.onStart(CordovaActivity.java:303)
                      at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1334)
                      at android.app.Activity.performStart(Activity.java:7019)
                      at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2741)
```

**Comment by JustinPierce** [comment]
@jeremy-dobler I'm seeing this problem as well, and it turns out that the same workaround I used when Intercom originally went to 4.x also works here.

The problem is that the Intercom plugin pulls down the latest version of the Android support library (currently `27.0.2`) while everything else in my build process wants `26.x`.

The workaround is to install [`cordova-plugin-gradle-support-release`](https://github.com/dpa99c/cordova-android-support-gradle-release). This plugin forces all of your Cordova plugins to use the same version of the support library, if they ask for it. As of today, it's defaulting to `26.+`, which should give you `26.1.0`. I have confirmed that using this version allows my project to launch again on Android, and that Intercom is fully functional.
